stoplight.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function() {
+
+    var light = $('.bulb');
+
+    var handleBulbClick = function() {
+        $(this).addClass('active')
```

Comment:
If you change this to `toggleClass`, it would take care of turning off an already-lit light if you click on it.